From 729410b4b0c9fc9a8d8462480ef580b5b97782d9 Mon Sep 17 00:00:00 2001
From: Wolfgang Bumiller <w.bumiller@proxmox.com>
Date: Thu, 10 Dec 2015 15:21:59 +0100
Subject: [PATCH 39/41] qapi: qmp_mashal_* renames for get_link_status

As per 7fad30f0:
qapi: Rename qmp_marshal_input_FOO() to qmp_marshal_FOO()

For-patch: qmp: add get_link_status
---
 qmp-commands.hx | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/qmp-commands.hx b/qmp-commands.hx
index 129e1b3..12b884c 100644
--- a/qmp-commands.hx
+++ b/qmp-commands.hx
@@ -1773,7 +1773,7 @@ EQMP
     {
         .name       = "get_link_status",
         .args_type  = "name:s",
-        .mhandler.cmd_new = qmp_marshal_input_get_link_status,
+        .mhandler.cmd_new = qmp_marshal_get_link_status,
     },
 
 SQMP
-- 
2.1.4

